.github/workflows/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,17 @@
+blank_issues_enabled: false
+contact_links:
+  - name: New Idea?
+    url: https://github.com/CommunityToolkit/Labs-Windows/discussions?discussions_q=category%3AExperiments+category%3AIdeas
+    about: Labs is where we propose and experiment with new ideas. Search ongoing experiments or contribute your own!
+  - name: Questions about the Toolkit?
+    url: https://github.com/CommunityToolkit/Windows/discussions/categories/questions-help
+    about: I have a question about how to use something in the toolkit.
+  - name: Win32 Controls
+    url: https://github.com/CommunityToolkit/Microsoft.Toolkit.Win32/issues/new
+    about: I have an issue with a Toolkit WPF or WinForms control
+  - name: MVVM Toolkit, High Performance, or Diagnostic Package? 
+    url: https://aka.ms/toolkit/dotnet
+    about: "If you have a question on these, see the .NET Community Toolkit repo:"
+  - name: Discord
+    url: https://aka.ms/wct/discord
+    about: "Join the UWP Discord Server and talk to us in the #community-toolkit channel."
```

Comment:
We probably want to link to the MAUI Community Toolkit and then also the WASDK and Uno Platform repos, eh?

Reply:
Sure, we can do that